.changeset/orange-books-fix.md

```diff
@@ -0,0 +1,5 @@
+---
+"create-t3-app": patch
+---
+
+Fixes fallback branch if init.defaultBranch is not set
```

Comment:
There should only be one changeset for a PR

Reply:
I've removed the changelog commit